### ISSUE TITLE
item can not resize after dragging between two grids

### ISSUE THREAD

**Issue by eagoo** [troolee/gridstack.js#416]
item can not resize after dragging between two grids in demo/two.html

**Comment by eagoo** [comment]
drop: function(event, ui) {
                    self.placeholder.detach();

                    var node = $(ui.draggable).data('_gridstack_node');
                    node._grid = self;
                    var el = $(ui.draggable).clone(false);
                    el.data('_gridstack_node', node);
                    $(ui.draggable).remove();
                    node.el = el;
                    self.placeholder.hide();
                    el
                        .attr('data-gs-x', node.x)
                        .attr('data-gs-y', node.y)
                        .attr('data-gs-width', node.width)
                        .attr('data-gs-height', node.height)
                        .addClass(self.opts.itemClass)
                        .removeAttr('style')
                        .enableSelection()
                        .removeData('draggable')
                        .removeClass('ui-draggable ui-draggable-dragging ui-draggable-disabled')
                        .unbind('drag', onDrag);
                  **  if(el.find('div.ui-resizable-handle').length >0){
                        el.find('div.ui-resizable-handle').remove()
                    }**


-------------------
drop function should judge the el has ui-resizable-hangle div and romve it
then next create new div for it

**Comment by doelgonzo** [comment]
+1

**Comment by doelgonzo** [comment]
The change suggested by @eagoo worked for me. Can we get this packaged in a version?

**Comment by eagoo** [comment]
#421

**Comment by griphi** [comment]
Hi,

the issue is sill in the demo/two.html file included.
If I am dragging between Grid 1 and Grid 2 the Widgets cannot be resized again.

Thanks

**Comment by radiolips** [comment]
Thanks for your help, @eagoo . We'll work on resolving this. For the time being, all work on this issue will be in #451 , which covers several related issues.

**Comment by Illizian** [comment]
A temporary fix for this (that doesn't require amends to library files) follows:

```
var grid = $('#grid').data('gridstack');

$('#grid').on('change', function(event, items) {
  items.forEach(function(item) {
    // Remove the old drag handles
    item.el[0].lastChild.remove();
  });
  // Re-Enable resize to re-attach handle listeners
  grid.enableResize(true, true);
});
```

**Comment by NaFreitaz** [comment]
I am still experiencing this issue, as is your demo: http://gridstackjs.com/demo/two.html.

Why has this been closed?

**Comment by JHoerbst** [comment]
Still occurring.

**Comment by radiolips** [comment]
As stated above @NaFreitaz, use #451 .

**Comment by JHoerbst** [comment]
@Illizian doesn't work  on me

**Comment by Illizian** [comment]
@JHoerbst I just ran the event listener code within the demo (http://gridstackjs.com/demo/two.html) and it worked perfectly.

- Try adding breakpoints within the callback
- Check the selector is correct for your circumstance (e.g `#grid` becomes `.grid-stack` for the demo above, perhaps you've a different id/class too?)
